crates/sui/src/unit_tests/snapshots/sui__upgrade_compatibility__upgrade_compatibility_tests__declarations_missing.snap

```diff
@@ -38,6 +38,16 @@ error[Compatibility E01001]: missing public declaration
   = structs are part of a module's public interface and cannot be removed or changed during an upgrade.
   = add missing struct 'StructToBeRemoved' back to the module 'struct_'.
 
-The following modules are missing from the new package: 'missing_module'
+error[Compatibility E01007]: module missing
+  ┌─ /Users/jordanjennings/code/sui/crates/sui/src/unit_tests/fixtures/upgrade_errors/declaration_errors_v2/Move.toml:1:1
+  │  
+1 │ ╭ [package]
+2 │ │ name = "upgrades"
+3 │ │ edition = "2024.beta" # edition = "legacy" to use legacy (pre-2024) Move
```

Comment:
the two options I considered were:
- write a simple pattern matching implementation to show this package table which is done here
- create a toml parser to generate Locs for '[package]' and 'name =' etc

The second would certainly be better and multiple uses but I think it would be good to get some buy in on the larger piece of work if we find this important and the toml sourcemap useful elsewhere.